src/sysinfo/nvapi_adapter.cpp

```diff
@@ -211,27 +228,39 @@ namespace dxvk {
         if (m_vkProperties.deviceID >= 0x2600)
             return NV_GPU_ARCHITECTURE_AD100;
 
+        // See https://gitlab.freedesktop.org/mesa/mesa/-/blob/main/src/nouveau/vulkan/nvk_physical_device.c
+        // for NVK properties and features
+
         // KHR_fragment_shading_rate's
         // primitiveFragmentShadingRateWithMultipleViewports is supported on
         // Ampere and newer
+        // TODO: We haven't found a way yet to identify Ampere on NVK
         if (IsVkDeviceExtensionSupported(VK_KHR_FRAGMENT_SHADING_RATE_EXTENSION_NAME)
             && m_vkFragmentShadingRateProperties.primitiveFragmentShadingRateWithMultipleViewports)
             return NV_GPU_ARCHITECTURE_GA100;
 
-        // Variable rate shading is supported on Turing and newer
-        if (IsVkDeviceExtensionSupported(VK_NV_SHADING_RATE_IMAGE_EXTENSION_NAME))
+        // Mesh shader is supported on Turing and newer
+        // This needs https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/27196 on NVK
+        if (IsVkDeviceExtensionSupported(VK_EXT_MESH_SHADER_EXTENSION_NAME))
             return NV_GPU_ARCHITECTURE_TU100;
 
-        // VK_NVX_image_view_handle is supported on Volta and newer
-        if (IsVkDeviceExtensionSupported(VK_NVX_IMAGE_VIEW_HANDLE_EXTENSION_NAME))
+        // VK_NVX_image_view_handle is supported on Volta and newer on the NVIDIA proprietary driver
+        // VK_EXT_depth_clip_control's depthClipControl is supported on Volta and newer on NVK
+        if ((HasNvProprietaryDriver() && IsVkDeviceExtensionSupported(VK_NVX_IMAGE_VIEW_HANDLE_EXTENSION_NAME))
+            || (HasNvkDriver()
+                && IsVkDeviceExtensionSupported(VK_EXT_DEPTH_CLIP_CONTROL_EXTENSION_NAME)
+                && m_vkDepthClipControlFeatures.depthClipControl))
             return NV_GPU_ARCHITECTURE_GV100;
 
         // VK_NV_clip_space_w_scaling is supported on Pascal and newer
+        // TODO: We haven't found a way yet to identify Pascal on NVK
```

Comment:
Actually, I think this is `Pascal` https://gitlab.freedesktop.org/mesa/mesa/-/blob/main/src/nouveau/vulkan/nvk_physical_device.c?ref_type=heads#L682 (also according to https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/28159/diffs#059d99b2edae5a4b51e79eb38f6f299f37f26831)

Reply:
It's Pascal indeed (MR has been merged now) and from a quick look at gpuinfo this appears to match what NV proprietary reports.